pkgs/tools/misc/kamp/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  replaceVars,
+  perl,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kamp";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "vbauerster";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9cakFhA9niMZ0jD0ilgCUztk4uL6wDp6zfHUJY/yLYw=";
+  };
+
+  cargoHash = "sha256-BnVV0UnXEebq1kbQvv8PkmntLK0BwrOcMxxIODpZrxc=";
+
+  postInstall = ''
+    install scripts/* -Dt $out/bin
+  '';
+
+  meta = {
+    description = "Tool to control Kakoune editor from the command line.";
+    homepage = "https://github.com/vbauerster/kamp";
+
+    license = with lib.licenses; [
+      unlicense
+    ];
+
+    maintainers = with lib.maintainers; [ erikeah ];
+
+    mainProgram = "kamp";
+  };
+}
```

```suggestion
})
```

Closes the suggested `finalAttrs`